common/src/main/java/org/valkyrienskies/mod/mixin/client/MixinCamera.java

```diff
@@ -128,6 +128,19 @@ private void setRotationWithShipTransform(final float yaw, final float pitch, fi
         this.left.transform(this.rotation);
     }
 
+    @Unique
+    public void addRotation(final Quaterniondc quaterniondc) {
```

Comment:
You should not add public functions to mixins, unless you are implementing an interface.